### ISSUE TITLE
Contact Form: A11Y - (required) label doesn't have enough contrast ratio with default opacity setting

### ISSUE THREAD

**Issue by stronenv** [Automattic/jetpack#31203]
### Impacted plugin

Jetpack

### Quick summary

The default opacity for the `(required)` label is `.45`. Using black text on a white background it does not pass the contrast ratio requirements for WCAG 2.1 at level AA, 4.5:1.

The text color varies from theme to theme, but suggesting changing the default opacity for the `(required)` label to `.6` to allow for a wider array of colors to pass the contrast ratio requirements. 

### Steps to reproduce

1. Insert a form block, and make a field required
2. The `(required)` label appears and is dimmed with `opacity: .45 `

### A clear and concise description of what you expected to happen.

The `(required)` label should pass the contrast ratio requirements for Normal Text WCAG 2.1 at level AA, 4.5:1. 

### What actually happened

With black text on a white background, it leaves the `(required)` label with a contrast of 3.4:1

### Impact

Most (> 50%)

### Available workarounds?

Yes, easy to implement

### Platform (Simple and/or Atomic)

Simple, Atomic, Self-hosted

**Comment by jeherve** [comment]
Related: #16685.
